app/scripts/lib/multichain-api/provider-authorize/handler.js

```diff
@@ -189,6 +189,7 @@ export async function providerAuthorizeHandler(req, res, _next, end, hooks) {
               value: {
                 requiredScopes: grantedRequiredScopes,
                 optionalScopes: grantedOptionalScopes,
+                isMultichainOrigin: true,
```

Comment:
should we maybe put this in `sessionProperties` to store it?

Reply:
🤔 maybe? So we'd add it to all permissions even if not present in the request? I suppose thats okay. Happy either way